projects/storefrontlib/src/recipes/config/b2b-occ-config.ts

```diff
@@ -9,6 +9,8 @@ export const defaultB2bOccConfig: OccConfig = {
         setDeliveryAddress:
           'orgUsers/${userId}/carts/${cartId}/addresses/delivery',
         placeOrder: 'orgUsers/${userId}/orders?termsChecked=true',
+        scheduleReplenishmentOrder:
+          'orgUsers/${userId}/replenishmentOrders?cartId=${cartId}&termsChecked=${termsChecked}',
```

Comment:
I don't think you need this part "cartId=${cartId}&termsChecked=${termsChecked}" since you are already adding it to your params in https://github.com/SAP/spartacus/pull/8504/files#diff-9aec5d768616f6e03649ddefca5f8616R30